lib/scorecard_api/configuration.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'common/client/configuration/rest'
+require 'common/client/middleware/response/scorecard_api_errors'
+require 'common/client/middleware/response/json_parser'
+require 'common/client/middleware/response/raise_error'
+require 'common/client/middleware/response/snakecase'
```

Comment:
These are unnecessary.

Reply:
So if remove them then try to fetch the data then messages like this happen
<img width="935" alt="Screen Shot 2020-01-17 at 3 35 35 PM" src="https://user-images.githubusercontent.com/1094999/72644503-2b46bb00-393f-11ea-94d4-9cc7389e9c0b.png">